lib/ansible/modules/cloud/amazon/ecs_ecr.py

```diff
@@ -43,6 +42,17 @@
             - if yes, remove the policy from the repository
         required: false
         default: false
+    lifecycle_policy:
+        description:
+            - JSON or dict that represents the new lifecycle policy
+        required: false
+        version_added: '2.5'
+    delete_lifecycle_policy:
```

Comment:
Prefer purge_lifecycle_policy to be similar to other modules.

Reply:
I'll need to also deprecate `delete_policy` and replace it with `purge_policy` for consistency.

Reply:
Fixed.